src/test/java/io/stargate/sgv2/jsonapi/fixtures/testdata/TestData.java

```diff
@@ -0,0 +1,55 @@
+package io.stargate.sgv2.jsonapi.fixtures.testdata;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TestData {
```

Comment:
Maybe it makes sense, but name "TestData" is bit too generic and can mess with IDE auto-completion. Maybe it could use a prefix like other XxxTestData classes do?

Reply:
yeah it's still a bit WIP, the idea was to share things like the names so they can be re-used